ee/codegen/src/generators/workflow-output.ts

```diff
@@ -12,19 +12,22 @@ export declare namespace WorkflowOutput {
   export interface Args {
     workflowContext: WorkflowContext;
     workflowOutputContext: WorkflowOutputContext;
+    outputNamesById: Map<string, string>;
   }
 }
 
 export class WorkflowOutput extends AstNode {
   private workflowContext: WorkflowContext;
   private workflowOutputContext: WorkflowOutputContext;
   private workflowOutput: Field;
+  private outputNamesById: Map<string, string>;
```

Comment:
We shouldn't need to pass/duplicate this here, right? We have the `workflowContext` above and can access this same info through that?